production/helm/loki/templates/_helpers.tpl

```diff
@@ -243,6 +243,9 @@ s3:
   {{- with .disable_dualstack }}
   disable_dualstack: {{ . }}
   {{- end }}
+  {{- with .storageClass }}
+  storage_class: {{ . }}
```

Comment:
i think this also needs to be defined in values.yaml?
https://github.com/grafana/loki/blob/c2a34d214ea146a05456c3625229c22146cea450/production/helm/loki/values.yaml#L342

Reply:
Good catch @ashwanthgoli. I've added it and rebased the PR.